applications/FemToDemApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -78,7 +78,7 @@ void AddCustomProcessesToPython(pybind11::module &m)
 
 	// Normalized Free Energy extrapolation to Nodes
 	class_<ComputeNormalizedFreeEnergyOnNodesProcess, ComputeNormalizedFreeEnergyOnNodesProcess::Pointer, Process>(m, "ComputeNormalizedFreeEnergyOnNodesProcess")
-		.def(init<ModelPart &, unsigned int>())
+		.def(init<ModelPart &, unsigned int, Parameters>())
```

Comment:
See, here you are using unsigned int instead of int. BTW, you don't need dimension, DOMAIN_SIZE should be defined on the ProcessInfo of the model part